services/src/datasets/external/netcdfcf/mod.rs

```diff
@@ -125,6 +124,34 @@ impl<D: GeoEngineDb> DataProviderDefinition<D> for NetCdfCfDataProviderDefinitio
     }
 }
 
+impl NetCdfCfDataProviderDefinition {
+    async fn _initialize(self: Box<Self>) -> crate::error::Result<NetCdfCfDataProvider> {
+        let mut pg_pool_builder = Pool::builder();
+        if self.metadata_db_config.schema == "pg_temp" {
+            // having more than one connection with `pg_temp` leads to different temp schemata being used
+            pg_pool_builder = pg_pool_builder.max_size(1);
+        }
+
+        let metadata_db = pg_pool_builder
+            .build(PostgresConnectionManager::new(
+                self.metadata_db_config.pg_config(),
+                NoTls,
+            ))
+            .await?;
+
+        migrate_database(&mut metadata_db.get().await?, &all_migrations()).await?;
```

Comment:
this migration appears in the logs like a regular database migration, would be nice to be able to see that it's for netcdf

> 2024-03-04T14:35:45.457001Z  INFO geoengine_services::contexts::migrations::database_migration:84: Current database version: Some("0001_initial")

Reply:
[ec5ae72](https://github.com/geo-engine/geoengine/pull/931/commits/ec5ae72286c58bb1293891d4d588246881e92607)